code/game/machinery/hologram.dm

```diff
@@ -261,10 +261,11 @@ Possible to do for anyone motivated enough:
 				return
 			if(usr.loc == loc)
 				var/list/callnames = list()
-				for(var/I in holopads)
-					var/area/A = get_area(I)
-					if(A)
-						LAZYADD(callnames[A], I)
+				for(var/obj/machinery/holopad/I in holopads)
+					if ( I.is_operational() )
+						var/area/A = get_area(I)
+						if(A)
+							LAZYADD(callnames[A], I)
```

Comment:
```suggestion
				for(var/obj/machinery/holopad/pad as anything in holopads)
					if (pad.is_operational())
						var/area/A = get_area(pad)
						if(A)
							LAZYADD(callnames[A], pad)
```
`as anything` since that list will always be holopads